### ISSUE TITLE
pip3 install pylibblkid error in Centos

### ISSUE THREAD

**Issue by zoumingzhe** [vojtechtrefny/pyblkid#3]
```shell
[root@zou ~]# pip3 install pylibblkid
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3 install --user` instead.
Collecting pylibblkid
  Using cached https://files.pythonhosted.org/packages/ab/1b/d53d091f8d646e997a33424847b93f8c2be37efecca799247296a355e171/pylibblkid-0.2.tar.gz
Requirement already satisfied: pkgconfig in /usr/local/lib/python3.6/site-packages (from pylibblkid)
Installing collected packages: pylibblkid
  Running setup.py install for pylibblkid ... error
    Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-nmqve0zm/pylibblkid/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-he2kvbss-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_ext
    building 'blkid' extension
    creating build
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/usr/include -I/usr/include/python3.6m -c src/pyblkid.c -o build/temp.linux-x86_64-3.6/src/pyblkid.o -Wall -Wextra -Werror
    src/pyblkid.c: In function 'Blkid_partition_types':
    src/pyblkid.c:376:5: error: implicit declaration of function 'blkid_partitions_get_name' [-Werror=implicit-function-declaration]
         while (blkid_partitions_get_name (idx++, &name) == 0) {
         ^
    src/pyblkid.c: In function 'PyInit_blkid':
    src/pyblkid.c:526:56: error: 'BLKID_SUBLKS_BADCSUM' undeclared (first use in this function)
         PyModule_AddIntConstant (module, "SUBLKS_BADCSUM", BLKID_SUBLKS_BADCSUM);
                                                            ^
    src/pyblkid.c:526:56: note: each undeclared identifier is reported only once for each function it appears in
    cc1: all warnings being treated as errors
    error: command 'gcc' failed with exit status 1

    ----------------------------------------
Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-nmqve0zm/pylibblkid/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-he2kvbss-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-nmqve0zm/pylibblkid/
```

**Comment by zoumingzhe** [comment]
I have installed libblkid and libblikid-dev
```shell
[root@zou ~]# yum list |grep blkid
Failed to set locale, defaulting to C
libblkid.x86_64                          2.23.2-65.el7_9.1             @updates
libblkid-devel.x86_64                    2.23.2-65.el7_9.1             @updates
libblkid.i686                            2.23.2-65.el7_9.1             updates
libblkid-devel.i686                      2.23.2-65.el7_9.1             updates
```

**Comment by vojtechtrefny** [comment]
I never tested this with libblkid 2.23, but this should be just a couple more *ifdef*s to make sure it works with older releases too.

**Comment by vojtechtrefny** [comment]
Closed via #7
